terravision

```diff
@@ -93,7 +93,7 @@ def cli():
     default="architecture",
     help="Filename for output diagram (default architecture.dot.png)",
 )
-@click.option("--format", default="png", help="File format (png/pdf/svg/json/bmp)")
```

Comment:
format is only for the draw command. You cannot draw json

Reply:
cool